packages/react-components/src/components/VideoGallery/LocalVideoCameraCycleButton.tsx

```diff
@@ -3,18 +3,18 @@
 
 import { IButtonStyles, IconButton } from '@fluentui/react';
 import React from 'react';
-import { OptionsDevice } from '@internal/react-components/src/components/DevicesButton';
+import { OptionsDevice } from '../..';
```

Comment:
avoid importing from `src/`of the folder the component is in
```suggestion
import { OptionsDevice } from '../DeviceButton`